tests/test_livepatch.py

```diff
@@ -72,46 +73,47 @@ def test_enable_livepatch_apt_get_options(self):
 
     def test_enable_livepatch_installs_snap(self):
         """enable-livepatch installs the livepatch snap if needed."""
-        self.setup_livepatch(installed=False)
+        self.setup_livepatch(enabled=False)
         process = self.script('enable-livepatch', self.livepatch_token)
         self.assertEqual(0, process.returncode)
         self.assertIn(
             'Installing the canonical-livepatch snap', process.stdout)
 
     def test_is_livepatch_enabled_true(self):
         """is-livepatch-enabled returns 0 if the service is enabled."""
+        self.setup_livepatch(installed=True, enabled=True)
         process = self.script('is-livepatch-enabled')
         self.assertEqual(0, process.returncode)
 
     def test_is_livepatch_enabled_false(self):
         """is-livepatch-enabled returns 1 if the service is not enabled."""
-        self.setup_livepatch(enabled=False)
```

Comment:
So here we are effectivelly calling setup_livepatch(enabled=None), aren't we changing the test a little bit? It still passes either way.

Reply:
yeah I think this can be reverted (or we could have 2 tests)